test/users.test.js

```diff
@@ -0,0 +1,45 @@
+const request = require('supertest'),
+  app = require('../app');
+
+describe('POST /user', () => {
+  it('signUp User', done => {
+    request(app)
+      .post('/users')
+      .send({ name: 'yesica', lastName: 'nava', email: 'yesica@wolox.co', password: 'shdfgs345*' })
+      .set('Accept', 'application/json')
+      .then(response => {
+        expect(response.statusCode).toBe(201);
+        done();
+      });
+  });
+  it('validate email', done => {
+    request(app)
+      .post('/users')
+      .send({ name: 'yesica', lastName: 'nava', email: 'yesica.co', password: 'shdfgs345*' })
+      .set('Accept', 'application/json')
+      .then(response => {
+        expect(response.statusCode).toBe(400);
+        done();
+      });
+  });
+  it('validate password', done => {
+    request(app)
+      .post('/users')
+      .send({ name: 'yesica', lastName: 'nava', email: 'yesica.co', password: 'err*' })
+      .set('Accept', 'application/json')
+      .then(response => {
+        expect(response.statusCode).toBe(400);
+        done();
+      });
+  });
+  it('validate required fields', done => {
+    request(app)
+      .post('/users')
+      .send({ name: '', lastName: '', email: '', password: '' })
```

Comment:
if you send `{ name: '', lastName: '', email: '', password: '' }` the parameters are being sent but not with a valid value. 
i think that in this case it would be {} that you want to send.